Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -282,6 +298,13 @@ linkRCStatus (driverId, merchantId) req@RCStatusReq {..} = do
       deactivateRC rc driverId
   return Success
 
+updateRC :: (Id Person.Person, Id DM.Merchant) -> UpdateRCReq -> Flow APISuccess
+updateRC (driverId, merchantId) UpdateRCReq {..} = do
+  (rc, now) <- processRC driverId merchantId rcNo
+  validateRCActivation driverId merchantId rc
```

Comment:
This function, will assign this RC to the driver too if it is not linked to this driver. Why update RC function need to take driverId at all?